core/unit_test/hip/TestHIP_InterOp_Graph.cpp

```diff
@@ -101,6 +101,9 @@ TEST(TEST_CATEGORY, graph_instantiate_and_debug_dot_print) {
 
   ASSERT_EQ(num_nodes, 2u);
 
+  // hipGraphDebugDotPrint was introduced in ROCm 5.5
+#if (HIP_VERSION_MAJOR > 5) || \
+    ((HIP_VERSION_MAJOR > 5) && (HIP_VERSION_MINOR >= 5))
```

Comment:
I tend to agree with @dalg24 that the whole test could be skipped in this case. Not blocking though.